CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # credential-handler-polyfill ChangeLog
 
+## 3.2.1 - 2024-05-dd
+
+### Fixed
+- Ensure `navigator.credentials[get|store]` methods from the polyfill
+  will continue to run to process `options.web|WebCredential` if other
+  software that runs after it is setup changes those methods.
```

Comment:
```suggestion
  will continue to run and process `options.web|WebCredential`, even
  if those methods are changed by other software that runs after the
  polyfill is set up.
```

Reply:
Yeah, I was thinking about fixing up that message on second read. Thanks for doing it for me, Ted!